academy/xl-run-prod/3-genesis.md

```diff
@@ -193,7 +196,7 @@ Because a validator can be configured with multiple flags, look at:
 $ ./myprojectd gentx --help
 ```
 
-Each validator then returns you their transaction file(s).
+Each validator then returns their transaction file(s).
```

Comment:
I think the "you" is important. Without it, it is not clear to whom they return it.

Reply:
I feel that the structure "X returns you Y" isn't typical in English (not sure if it's technically _wrong_ or not), but I notice that line 203 states "a validator returns a signed transaction **_to you_**" - is it enough for us to be explicit there?

Reply:
I think we need to be explicit on line 199 too because 203 is "too far", with the heading.

Reply:
I've made a change to read "Each validator then returns **to you** their transaction file(s)." It has a slightly _formal_ feel to it, but I think it reads better than "returns you" in terms of the English. :-)